WeatherPage.xaml

```diff
@@ -9,21 +9,19 @@
     <ScrollView>
         <ScrollView>
             <VerticalStackLayout
-            Spacing="25"
-            Padding="30,0"
             VerticalOptions="Start">
                 <!-- This line says that the ItemsSource of the CollectionView is the Weathers property of the Domain object (an ObservableCollection) -->
-                <CollectionView x:Name="CV" ItemsSource="{Binding Weathers}" SelectionMode="Single">
+                <CollectionView x:Name="CV" ItemsSource="{Binding ClosestAirportWeather}" SelectionMode="Single">
```

Comment:
Related to the above, if this has only one Airport, it doesn't need a CollectionView. I presume you're just using this as a simple way to organize several labels, this would be better off as a VerticalStackLayout.

Upon reflection, it _would_ be better to leave an Entry for an airport.  Let's make the UI look something like the following, where Airport, METAR and TAF are int he same font size, and a Fetch Weather button appears to the right of the Entry. 


Airport: [ KATW ] [Fetch Weather]

METAR:
=================


TAF:
=================